### ISSUE TITLE
This repo doesn't know it's a fork

### ISSUE THREAD

**Issue by benjaminrosenbaum** [dchealthlink/MITC#1]
lincolnsmithy/medicaid_eligibility knows that it's forked from https://github.com/HHSIDEAlab/medicaid_eligibility, so it can easily make pull requests, but dchealthlink/MITC seems to have lost that connection. @tiggrdave, @lincolnsmithy -- do you know why?

**Comment by lincolnsmithy** [comment]
Yes, I pushed this code to dchealthlink from our "local- with our fixes/enhancements".  Looks like I need to remote add upstream HHSIDEAlab repo to re-sync - will come find you to confirm/research a bit more.
Thanks,
